packages/media/src/webrtc/VegaRtcManager.ts

```diff
@@ -256,6 +256,9 @@ export default class VegaRtcManager implements RtcManager {
             roomName: this._room.name,
             eventClaim: this._room.isClaimed ? this._eventClaim : null,
             lowBw: "true",
+            ...Object.keys(this._features || {})
+                .filter((featureKey) => this._features[featureKey] && /^sfu/.test(featureKey))
```

Comment:
Do we need the check for `this._features[featureKey]`?

Reply:
just for filtering out features that are false/off